src/lists/ListItem/ListItem.jsx

```diff
@@ -1,44 +1,36 @@
 import PropTypes from 'prop-types';
 
-import CSS from './styles.css'
-import Heading from '../../typography/Heading'
-import Text from '../../typography/Text'
-import Toggle from '../../forms/Toggle'
+import CSS from './styles.css';
+import Heading from '../../typography/Heading/Heading';
+import Text from '../../typography/Text/Text';
+import Toggle from '../../forms/Toggle/Toggle';
 
-const ListItem = ({
-	title,
-	icon,
-	iconColor,
-	description,
-	className,
-	onLink,
-	linkText,
-	onChange,
-	checked,
-	dataTestId,
-}) => (
+const ListItem = ({ title, icon, iconColor, description, className, onLink, linkText, onChange, checked, dataTestId }) => (
 	<li className={`${CSS.item} ${className}`} data-testid={dataTestId}>
-		{ (icon) ? <div data-testid={`${dataTestId}-icon`} className={`${CSS.media} ${ (iconColor ? CSS[iconColor] : '')}`}> {icon} </div> : null }
+		{icon ? (
+			<div data-testid={`${dataTestId}-icon`} className={`${CSS.media} ${iconColor ? CSS[iconColor] : ''}`}>
+				{' '}
+				{icon}{' '}
+			</div>
+		) : null}
 		<div data-testid={`${dataTestId}-heading`} className={CSS.body}>
 			<Heading type="txt-medium">{title}</Heading>
-			{ (description) ? <Text dataTestId={`${dataTestId}-description`}>{description}</Text> : null}
-			{ (onLink) ? (
+			{description ? <Text dataTestId={`${dataTestId}-description`}>{description}</Text> : null}
+			{onLink ? (
 				<span
 					data-testid={`${dataTestId}-heading-link`}
 					onClick={() => {
-						onLink()
+						onLink();
```

Comment:
This could be used as a direct prop
```suggestion
					onClick={onLink}
```